package.json

```diff
@@ -50,7 +50,8 @@
   },
   "scripts": {
     "start": "npm run re:build && react-scripts start",
-    "build": "npm run re:build && EXTEND_ESLINT=true react-scripts build",
+    "build-js-only": "GENERATE_SOURCEMAP=false EXTEND_ESLINT=true react-scripts build --verbose",
```

Comment:
I think I should also remove this `GENERATE_SOURCEMAP=false`, they are useful for debugging in production. I only added it when the builds were taking so long.